### PR TITLE
gen_rs: pass through ForeignMod to the CXX bridge for now

### PR DESCRIPTION
**Pull request by ahayzen-kdab**
Later we will pass through all non CXX-Qt items to the CXX bridge,
but this is a stepping stone that allows us to progress.

### CODE REVIEW

**Comment by ahayzen-kdab** [comment]
This doesn't work with `unsafe extern "C++"` ...

**Comment by ahayzen-kdab** [comment]
> This doesn't work with `unsafe extern "C++"` ...

So `extern "T"` comes through as `ItemForeegnMod` but `unsafe extern "T"` comes through as `Verbatim`.

**Comment by ahayzen-kdab** [comment]
> > This doesn't work with `unsafe extern "C++"` ...
> 
> So `extern "T"` comes through as `ItemForeegnMod` but `unsafe extern "T"` comes through as `Verbatim`.

This works now.

Just need to also support attributes on the extern block.

**Comment by ahayzen-kdab** [comment]
> Just need to also support attributes on the extern block.

This is working now

File: cxx-qt-gen/test_inputs/passthrough.rs [hunk 1]
```diff
@@ -21,6 +21,22 @@ pub mod my_object {
     // ItemForeignMod
     extern "C" {}
 
+    #[namespace = "namespace"]
+    extern "C" {}
+
+    #[namespace = "namespace"]
+    #[namespace = "namespace"]
```
Comment (Be-ing) [inline cxx-qt-gen/test_inputs/passthrough.rs:28]: Repeating the same name twice? Are you checking here that two attributes on consecutive lines are properly handled? If so, I think using two different attributes would make the test easier to understand.

Comment (ahayzen-kdab) [reply cxx-qt-gen/test_inputs/passthrough.rs:28]: Yup, i'm checking that multiple attributes are passed through, i'll rename one of them.

Comment (ahayzen-kdab) [reply cxx-qt-gen/test_inputs/passthrough.rs]: Fixed.

**Review by Be-ing — APPROVED** [review]
waiting for CI

**Comment by ahayzen-kdab** [comment]
(Rebased as the other one landed)

**Review by Be-ing — APPROVED** [review]

**Pull request merged**